### PR TITLE
Adds support for nonPublicApi additional property

### PR DESCRIPTION
**Pull request by shadowsheep1**
I added support for nonPUblicApi additional property so that by default models are `public` and if you want them to be `internal` you have to setup the additional property `nonPubliApi` to `true`.

Fixes https://github.com/icerockdev/moko-network/issues/22

Reference: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/kotlin-client/api.mustache

### CODE REVIEW

File: sample/mpp-library/build.gradle.kts [hunk 1]
```diff
@@ -41,4 +41,5 @@ dependencies {
 openApiGenerate {
     inputSpec.set(file("src/swagger.json").path)
     generatorName.set("kotlin-ktor-client")
+    additionalProperties.set(mutableMapOf("nonPublicApi" to "true"))
```
Comment (Alex009) [inline sample/mpp-library/build.gradle.kts]: i think we should set this property to true as default. for those devs that just update version of library in own projects, they should not got public api by default if all was internal later

Comment (Alex009) [reply sample/mpp-library/build.gradle.kts]: can you do it, or you need help?

Comment (shadowsheep1) [reply sample/mpp-library/build.gradle.kts]: It depends if you wanna follow the _openapi client gen_ "standard" that has this option `false` by defalut or you prefer to diverge from it to preserver backward-compatibility (and you're saing that). I can have a look, but in these days I'm pretty busy so it'll take longer [-;

Comment (shadowsheep1) [reply sample/mpp-library/build.gradle.kts]: @Alex009 I've found 10 minutes this evening to look at it. Btw, this library behaves with a negative logic if you are familiar with openapitools client code gen.

**Comment by shadowsheep1** [comment]
Hi @Alex009 with my last commit, now this PR is backward compatible [-;

**Comment by Alex009** [comment]
thanks!

**Pull request merged**